recipes/isofit/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "isofit" %}
+{% set version = "1.1.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/adam-erickson/{{ name }}/archive/v{{ version }}.tar.gz
```

Comment:
@adam-erickson AFAICT the canonical source url for isofit is not your personal Github account. It is 
```
https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```
... which points to the official release.
This was all captured over on https://github.com/conda-forge/staged-recipes/pull/8940 when I initially posted the recipe.
It would be great if the efforts could align and there is one way of doing this.
wdyt?

Reply:
Hi @lewismc, I hope I have not stepped on any toes. I pointed the conda recipe to a release in my fork as a temporary solution until the official `isofit/isofit` repository is updated. It was merely meant to get the ball rolling for our team.

Reply:
Hi @adam-erickson absolutely not. I think what your doing is great I just wanted to make sure the recipe reflected something which will be in place in the longer term. 
From my experience, significant changes to recipes (such as the canonical source artifact url changing) is not particularly well received from the conda-forge maintainers and should be avoided if possible. 
I am not particularly aligned with isofit right now so have no control over what you end up doing as a solution however I did want to just drop in the above comment to make you aware of the other recipe. 
I'll go ahead and close off my PR right now, it can always be opened at a later stage.

Reply:
Conda-forge has a strong preference for patch files being applied to an official source rather than pulling the source from a fork ([example](https://github.com/conda-forge/clangdev-feedstock/blob/master/recipe/meta.yaml#L11-L18)). It's slightly harder at first but it makes long term maintenance and handling updates much easier.

Reply:
Thank you for the heads up. We will point the conda recipe to the PyPi package source URL or the release URL on the official GitHub page for `isofit`.